src/Certify.Core/ACMESharpCompat/Helpers.cs

```diff
@@ -31,6 +32,15 @@ public static AcmeClient GetClient(VaultInfo Config)
             var p = Config.Proxy;
             var _Client = new AcmeClient();
 
+            // depends on https://github.com/ebekker/ACMESharp/pull/300
+            //_Client.UserAgent = $"Certify/{0} {_Client.UserAgent}";
+            //_Client.Language = "en-US, en;q=0.8";
+            _Client.BeforeGetResponseAction = req =>
+            {
+                var version = typeof(ClientHelper).Assembly.GetName().Version;
+                req.UserAgent = $"Certify/{version} {req.UserAgent}";

```

Comment:
This should fix #160 also. 

I found a hook to get access to the Http Request and set the headers there, although the PR (https://github.com/ebekker/ACMESharp/pull/300) is still open to do it "in the open"